launcher.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/python3
```

Comment:
I'd like some brief description of the intention and scope of this script. What it adds on top of the `mix` commands it runs

Also make reference to it and explain in `README.md` - an indication when should I use this and when should I rather use Vagrant and when should I just run the `mix` commands in shell would be useful.

If the sole intended client of `launcher.py` is the `Dockerfile`, I'd rather have all these files in a subdir `docker` or sth.

Reply:
It adds nothing to the `mix` commands. It executes them to enable start up of the service. At the moment we cannot start the chain manually, nor can we recover from a failed chain. This is a work in progress, and as such the `README` should not be updated. I'm checking in smaller PRs so that you don't have a PR from hell to review.

I'm not sure on the benefit of moving `launcher.py` to a subdir. Additionally, it will require more code to be written to prevent path problems when trying to execute code.

Reply:
if it adds nothing, except a convenience to just do everything in one `Dockerfile` line, then maybe we should consider this temporary and look towards substituting this with `Mix.Tasks` which we have somewhere in plans. Unless python specifically will provide you with some tools not available from Mix.

the subdir benefit would be hiding that and marking it is used specifically by the docker setup. But if that's cumbersome then ok.